protos/brbpb/brbpb.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package brbpb;
+
+option go_package = "github.com/filecoin-project/mir/pkg/pb/brbpb";
+
+import "mir/codegen_extensions.proto";
+import "net/codegen_extensions.proto";
+
+// ============================================================
+// Events
+// ============================================================
+
+message Event {
+  option (mir.event_class) = true;
+
+  oneof type {
+    option (mir.event_type) = true;
+
+    BroadcastRequest request = 1;
+    Deliver          deliver = 2;
+  }
+}
+
+message BroadcastRequest {
+  option (mir.event) = true;
+
+  int64 id   = 1;
+  bytes data = 2;
+}
+
+message Deliver {
+  option (mir.event) = true;
+
+  int64 id   = 1;
+  bytes data = 2;
+}
+
+// ============================================================
+// Net Messages
+// ============================================================
+
+message Message {
+  option (net.message_class) = true;
+
+  oneof type {
+    option (net.message_type) = true;
+
+    StartMessage start_message = 1;
+    EchoMessage echo_message = 2;
+    ReadyMessage ready_message = 3;
+  }
+}
```

Comment:
It would be better to separate the implementation (in this case, Bracha's double-echo protocol) from the abstraction (brb)